saleor/discount/models.py

```diff
@@ -22,8 +22,7 @@ class NotApplicable(ValueError):
 
 class VoucherQueryset(models.QuerySet):
 
-    def active(self):
-        today = date.today()
+    def active(self, today):
```

Comment:
If we can now pass any date in the past, does it make sense to call it `today`?

Reply:
True, it is possible. But checking date in the past except in tests had no sens in our app. Changed.